src/languages/swift/index.ts

```diff
@@ -110,9 +111,7 @@ const swiftLanguage: Language = {
       [
         ["mul", "*"],
         ["div", "/"],
-        ["trunc_div", "/"],
         ["mod", "%"],
-        ["rem", "%"],
```

Comment:
Checking on code.golf, I think it's the other way around, Swift actually has `trunc_div` and `rem` and no `div` and `mod`.

Reply:
right, whoops